grpc-api/server/native/build.gradle

```diff
@@ -22,14 +23,45 @@ dependencies {
   // runtimeOnly project(':log-to-logbuffer')
 }
 
-application {
+distributions {
```

Comment:
With the removal of the grpc-api-server-docker, it probably makes sense to move this up to grpc-api-server itself?

I'm content to have this change happen later, probably in the jetty branch where I'm renaming grpc-api to server anyway, and adding jetty vs netty variants, but just want to be sure this makes sense as a plan.

Reply:
Yeah, I'd support a rename.